scripts/duplicate_pr.py

```diff
@@ -0,0 +1,124 @@
+from enum import Enum
+
+import argparse
+import sh
+
+
+class BranchName(Enum):
+    MASTER = "master"
+    FORMPLAYER = "formplayer"
+
+
+def get_git(path=None):
+    return sh.git.bake(_tty_out=False, _cwd=path)
+
+
+def get_target_branch(orig_target_branch:str):
+    if orig_target_branch == BranchName.FORMPLAYER.value:
+        return BranchName.MASTER.value
+    elif orig_target_branch == BranchName.MASTER.value:
+        return BranchName.FORMPLAYER.value
+    else:
+        print("Only following branches are allowed: {}".format(", ".join([key.value for key in BranchName])))
+        exit(1)
+
+
+def git_create_branch(orig_branch_name:str, new_branch_name: str, git=None):
+    git = git or get_git()
+    try:
+        git.checkout(orig_branch_name)
```

Comment:
Won't this error if the branch doesn't exist locally? I think you need to call `git_fetch_pr_branch` before this is called

Reply:
yes fair point, I was assuming part of the user set up would be making sure latest master/formplayer is pulled but this change improves that.
[0e71243](https://github.com/dimagi/commcare-core/pull/1254/commits/0e71243aa392edb24de1c1960b4f8c5057bb390f)